### PR TITLE
chore(cash-in): add USDT and USDC as cash-in tokens

### PR DESCRIPTION
**Pull request by jh2oman**
We have added support for the following:

USDC: Simplex, Ramp, Moonpay, CBPay, Bitmama, Fonbnk
USDT: Simplex, Ramp, Moonpay, CBPay, Bitmama

https://linear.app/valora/issue/ACT-1015/update-address-metadata-for-usdcandt-cico

Will test on Testflight

### CODE REVIEW

**Review by cajubelt — APPROVED** [review]

File: src/data/mainnet/ethereum-tokens-info.json [hunk 1]
```diff
@@ -36,7 +37,9 @@
     "address": "0xdac17f958d2ee523a2206206994597c13d831ec7",
     "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/USDT.png",
     "infoUrl": "https://www.coingecko.com/en/coins/tether",
-    "minimumAppVersionToSwap": "1.72.0"
+    "minimumAppVersionToSwap": "1.72.0",
+    "isStableCoin": true,
```
Comment (jophish) [inline src/data/mainnet/ethereum-tokens-info.json:41]: Are there any side effects of marking these with `isStableCoin`? IIRC we don't do this for anything but the c-stables at the moment. Wondering if there are assumptions in the app that `isStableCoin: true` implies that the coin is a c-stable.

Comment (jh2oman) [reply src/data/mainnet/ethereum-tokens-info.json:41]: We already have it for USDC actually

Comment (jh2oman) [reply src/data/mainnet/ethereum-tokens-info.json:41]: The only current use I can find in the wallet is in [TokenDetails](https://github.com/valora-inc/wallet/blob/main/src/tokens/TokenDetails.tsx#L99)

We hide the +/- price differentials for stables

Comment (jh2oman) [reply src/data/mainnet/ethereum-tokens-info.json:41]: and other older versions of the wallet won't get the ethereum tokens anyways

Comment (jophish) [reply src/data/mainnet/ethereum-tokens-info.json:41]: ohhh okay, cool -- somehow missed we already had it on USDC.

**Review by jophish — APPROVED** [review]

**Pull request merged**